desktop/core/src/desktop/js/onePageViewModel.js

```diff
@@ -446,6 +446,10 @@ class OnePageViewModel {
       self.loadApp('importer');
       self.getActiveAppViewModel(vm => {
         vm.createWizard.source.path(path);
+        vm.currentStep(2);
+        vm.createWizard.source.interpreter('hive');
+        vm.createWizard.destination.name('default.xyz1');
+        window.setTimeout(function () {vm.createWizard.indexFile()}, 1000);
```

Comment:
One problem in this, table name and database name are fixed so any option how to get the current database name of left assist @romainr @JohanAhlen

Reply:
You can get it via huePubSub...

```
import { ASSIST_GET_DATABASE_EVENT} from 'ko/components/assist/events';

huePubSub.publish(ASSIST_GET_DATABASE_EVENT, {
  connector: theConnector,
  callback: databaseDef => {
    console.log(databaseDef)
  }
});
```